### ISSUE TITLE
Component narrative not rendering under comonents for freedonia-compliance project

### ISSUE THREAD

**Issue by Rikhter** [opencontrol/compliance-masonry#338]
Hello Open Control Github,

I've recently been using compliance-masonry trying to get an understanding of the kind of output it creates and I've been having a hard time rendering content from component yaml definitions.

My assumption is that naratives, verifications, and references for components and their coverage of standard controls for the selected certification are all included into the output?

I've tried my own test in which it is not producing any component information. In my attempt to indentify what I might be doing wrong I've tried leveraging the freedonia-compliance project. Which unfortunately leads to similar results.

I've checked out the master branch of freedonia-compliance project found [here](https://github.com/opencontrol/freedonia-compliance).
All I can get is the Audit Policy (a component) page rendered like so:
![image](https://user-images.githubusercontent.com/3032766/52829798-1a356e80-311e-11e9-9637-8e04ea681aec.png)
It's missing a bunch of narrative content, but it does successfully produce the reference link on the page. I've not yet tried other content as verification information yet.

Details:

- Operating on Windows
- Using the 64 bit version 1.1.5 as distributed from the zip underthe releases page for this repo.
- Gitbook version: 3.2.3 / CLI version: 2.3.2
- freedonia-compliance project as cloned from master

Steps to recreate:

- Download compliance-masonry -> store executables in `<somelocation>\compliance`
  - I've not installed it onto my path but am using it from a sibling folder to the project directory minor details but just in case.
- Install gitbook via npm `npm install -g gitbook-cli`
- Clone freedonia-compliance to `<somelocation>\freedonia-compliance`
- from freedonia-compliance run: `..\compliance\compliance-masonry get`
- run: `..\compliance\compliance-masonry docs gitbook  FredRAMP-low`
- run: `gitbook serve .\exports`

If you could let me know if I'm crazy - or if there is something wrong here?
In either case happy to help.

Cheers,
Tom

**Comment by Rikhter** [comment]
PS: For fear of creating issues, rather than talking to the community is there a Slack or other comms channel for opencontrol contributors / conumers?

**Comment by JJediny** [comment]
There is a slack channel you can join here https://opencontrol-slack-inviter.herokuapp.com/

**Comment by Rikhter** [comment]
As a follow up I've also run through this same proces using the linux rpm on Centos 7.
Other differences in that setup was that I was able to leverage everything off the path as opposed to having direct filesystem references.

Unfortunately result was the same.

**Comment by gregelin** [comment]
Freedonia-compliance has been updated to have all components within the repo (e.g., no dependencies) and tested to work with Compliance-Masonry v1.1.5. 

When I ran "* from freedonia-compliance run: `..\compliance\compliance-masonry get`" I received a report of a duplicate key because both "standards" and "certifications" referenced same dependency repo. (Note:  I would have thought that would not cause a problem, because they are different files.) I am not sure if that was your issue.

In any case, Freedonia-compliance needed an update anyway.
